addons/mail/static/src/messaging/component/notification_group/notification_group.js

```diff
@@ -0,0 +1,76 @@
+odoo.define('mail.messaging.component.NotificationGroup', function (require) {
+'use strict';
+
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component } = owl;
+
+class NotificationGroup extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const group = this.env.entities.NotificationGroup.get(props.notificationGroupLocalId);
+            return {
+                group: group ? group.__state : undefined,
+            };
+        });
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {mail.messaging.entity.Notification}
+     */
+    get group() {
+        return this.env.entities.NotificationGroup.get(this.props.notificationGroupLocalId);
+    }
+
+    /**
+     * @returns {string|undefined}
+     */
+    image() {
+        if (this.group.notification_type === 'email') {
+            return '/mail/static/src/img/smiley/mailfailure.jpg';
+        }
+    }
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClick(ev) {
+        this.group.openDocuments();
+    }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickFailureDiscard(ev) {
+        // avoid opening documents due to the click handler on the group itself
+        ev.stopPropagation();
```

Comment:
Shouldn't we actually do the reverse, a bit similar to what we do for global events, that is checking inside `_onClick` that the target is not the failure button?

Reply:
don't like `ev.stopPropagation()` whatsoever, so I'm for an alternative solution that lets event propagate.

Reply:
Done, this is actually pretty simple and could be done everywhere to get rid of all `stopPropagation` without any functional change.